Project.toml

```
@@ -0,0 +1,26 @@
+name = "ImageInTerminal"
+uuid = "d8c32880-2388-543b-8c61-d9f865259254"
+version = "0.4.0"
+
+[deps]
+ColorTypes = "3da002f7-5984-5a60-b8a6-cbb66c0b333f"
+Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
+ImageCore = "a09fc81d-aa75-5fe9-8630-4744c3626534"
+ImageTransformations = "02fcd773-0e25-5acc-982a-7f6622650795"
+
+[compat]
+ColorTypes = ">= 0.3.2"
+Crayons = ">= 0.1.0"
+ImageCore = ">= 0.7"
+ImageTransformations = ">= 0.3.0"
+julia = ">= 0.7"
+
```

I think we'll need this to make the test-time dependencies work

```suggestion
[targets]
test = ["Test", "FixedPointNumbers", "TestImages", "OffsetArrays", "CoordinateTransformations", "Rotations", "ImageMagick"]
```